.github/workflows/tests_linux.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       matrix:
         os: [ubuntu-22.04]
         pl_backend: ["lightning_qubit"]
-    timeout-minutes: 30
+    timeout-minutes: 60
```

Comment:
Just for my knowledge, why did you need 60min timeout? :thinking:

Reply:
30 minutes is just a bit short now that we have native mcm tests. The one-by-one shot implementation is unfortunately rather inefficient for more than ~1000 shots and we do need more than that to get even 1 or 2-digit matching on all parametrized tests.